src/Builder.js

```diff
@@ -34,6 +67,9 @@ Builder.prototype = {
     var terms = [];
     for (var idx = 0; idx < arguments.length; idx++) {
       var arg = arguments[idx];
+      if (!(arg instanceof pexprs.PExpr)) {
```

Comment:
Only the constructor function knows what the arguments are. Since not every argument is `PExpr` but some are just plain values, `fromRecipe` cannot do a reconstruction.
What would it reconstruct anyway? Builder methods call other builder methods to reconstruct their arguments/sub-PExprs.